playground/AWS/Frontend/Program.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Amazon.SimpleNotificationService;
+using Amazon.SQS;
+using Frontend.Components;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.Services.AddAWSService<IAmazonSQS>();
+builder.Services.AddAWSService<IAmazonSimpleNotificationService>();
+
+// Configuring messaging using the AWS.Messaging library.
+builder.Services.AddAWSMessageBus(messageBuilder =>
+{
+    // Get the SQS queue URL that was created from AppHost and assigned to the project.
+    var chatTopicArn = builder.Configuration.GetSection("AWS:Resources")["ChatTopicArn"];
```

Comment:
```suggestion
    var chatTopicArn = builder.Configuration["AWS:Resources:ChatTopicArn"];
```

This is one of the things we're going to have to wrangle with aspire. Passing connection information in a consistent way and consuming it in a consistent way is part of the opinionated stack and what makes it easy to use.

The low-level infrastructure is fine, but I hope we can build something a bit more like the other components use a single config section with a fixed schema per component.

Reply:
Updated the code.

The configuration section will be hard to have a fixed schema since the output of a CloudFormation stack is dynamic to whatever the user defines in the template.

Reply:
That breaks the ease of use in the application code. We want free form config to work as well. but part of the value of aspire beyond the orchestration is having a component on the other side that picks up this configuration and does something with it.

```C#
var builder = DistributedApplication.CreateBuilder(args);

var db = builder.AddMySql("server").AddDatabase("db");

builder.AddProject<Projects.WebApplication1>("api")
    .WithReference(db);

builder.Build().Run();
```

```C#
using Microsoft.EntityFrameworkCore;

var builder = WebApplication.CreateBuilder(args);

builder.AddServiceDefaults();
builder.AddMySqlDbContext<AppDb>("db"); // pick up the db configuration

var app = builder.Build();

app.MapGet("/", async (AppDb db) =>
{
    await db.Database.EnsureCreatedAsync();

    db.Entries.Add(new Entry { Name = Guid.NewGuid().ToString() });
    await db.SaveChangesAsync();

    return await db.Entries.ToListAsync();
});

app.Run();
```